src/appkit.cs

```diff
@@ -19583,6 +19595,14 @@ interface NSToolbarItem : NSCopying, NSMenuItemValidation, NSValidatedUserInterf
 		[Mac (11, 0), iOS (14, 0)]
 		[Export ("navigational")]
 		bool Navigational { [Bind ("isNavigational")] get; set; }
+
+#if __MACCATALYST__
+		[NoMac]
+		[MacCatalyst (13, 0)]
```

Comment:
No need for the MacCatalyst attr. No need for the #if and if we use an #if we do not need the NoMac.

Reply:
I tried removing this, and we started generating the bit completely on catalyst.

Reply:
I created an issue for this: https://github.com/xamarin/xamarin-macios/issues/12824